html/dom/documents/dom-tree-accessors/nameditem-fetch-01.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>Named items: fetch text/html</title>
+<link rel="author" title="bzbarsky" href="mailto:bzbarsky@mit.edu">
+<link rel="author" title="ExE Boss" href="https://ExE-Boss.tech">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#dom-htmldocument-nameditem">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+test(function() {
+  var url = 'data:text/html,<html xmlns="http://www.w3.org/1999/xhtml"><form name="x"/></html>';
+  var xhr = new XMLHttpRequest();
+  xhr.open("GET", url);
+  xhr.responseType = "document";
+  xhr.onload = function() {
+    var doc = xhr.responseXML;
+    assert_equal(doc.x, doc.getElementsByTagName("form"));
```

Comment:
Isn't it assert_equals? Did you run these tests?

Reply:
I did run these tests, and I’m pretty sure that it is `assert_equal`.

Reply:
https://web-platform-tests.org/writing-tests/testharness.html

Reply:
I&nbsp;just figured&nbsp;out&nbsp;why this&nbsp;wasn't&nbsp;working&nbsp;right, it’s&nbsp;because I&nbsp;used&nbsp;`test(…)` instead&nbsp;of&nbsp;`promise_test(…)`.